src/BasicMath/Method.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace SilverStripe\MFA\BasicMath;
```

Comment:
I'm kind of unsure about whether this should exist in the main package. I'd be fine with putting it in the tests directory, or a separate module, but maybe not in the distributable code section

Reply:
Yep I agree. I was talking about this with @Cheddam yesterday. It seems fine to just toss it in here for now - but I recommend we move it to documentation once we get closer to stabilising. Just seemed like a waste not to commit it. Happy to remove it if you feel strongly.